### PR TITLE
Patch 1 (WIP - Untested & Unfinished)

### PR DESCRIPTION
**Pull request by tkcjesse**
# **Untested**

Sorry for the large number of commits. I did it in the browser because I'm lazy right now. Also, it allows me to see my comments on individual files.

Just be sure to squash and merge.

### CODE REVIEW

File: life_server/Functions/Economy/fn_adjustPrices.sqf [hunk 1]
```diff
@@ -54,42 +52,42 @@ if (_factor isEqualTo 0) exitwith {};//the factor 0 is not a real group
...
 {
-    _resource = _x select 0;
-    _buyprice = _x select 1;
-    _sellprice = _x select 2;
-    _varprice = _x select 3;
-    _minprice = _x select 4;
-    _maxprice = _x select 5;
-    _changeco = 0.3 + random 1.4;
-    _x set [7, _sellprice];
+    private _resource = _x select 0;
```
Comment (BoGuu) [inline life_server/Functions/Economy/fn_adjustPrices.sqf:58]: PARAMS!

Comment (tkcjesse) [inline life_server/Functions/Economy/fn_adjustPrices.sqf]: @Boguu you can param in a forEach like that? yeah?

**Comment by dbenson24** [comment]
@tkcjesse just let me know when you're done making changes

**Comment by tkcjesse** [comment]
Yeah will do. Still working...

**Comment by tkcjesse** [comment]
I need to redo this... Don't wait on me lol

**Pull request closed without merging**